k8s/upgrade/src/bin/upgrade-job/common/kube_client.rs

```diff
@@ -46,12 +50,14 @@ pub(crate) struct KubeClientSet {
     pods_api: Api<Pod>,
     namespaces_api: Api<Namespace>,
     deployments_api: Api<Deployment>,
+    customresourcedefinition_api: Api<CustomResourceDefinition>,
```

Comment:
crd is well known, will be easier to read imho

```suggestion
    crd_api: Api<CustomResourceDefinition>,
```